File: D108/src/main/java/com/example/model/dto/UserDto.java
package com.example.model.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@NoArgsConstructor
public class UserDto {
    private int userId;
    private String email;
    private String id;
    private String password;
    private String nickname;
    private int point;
    private int gamesWon;
    private int totalGames;
    private int level;
    private int exp;
    private String createdAt;

    public UserDto(String email, String id, String password, String nickname) {
        this.email = email;
        this.id = id;
        this.password = password;
        this.nickname = nickname;
    }

    @Override
    public String toString() {
        return "User{" +
                "userId=" + userId +
                ", email='" + email + '\'' +
                ", id='" + id + '\'' +
                ", password='" + password + '\'' +
                ", nickname='" + nickname + '\'' +
                ", point=" + point +
                ", gamesWon=" + gamesWon +
                ", totalGames=" + totalGames +
                ", level=" + level +
                ", exp=" + exp +
                ", createdAt='" + createdAt + '\'' +
                '}';
    }
}

File: GameTset/app/src/main/java/com/example/gametset/room/model/dto/UserDto.kt
package com.example.gametset.room.model.dto

data class UserDto(
    val userId: Int,
    val id: String,
    val password: String,
    val nickname: String,
<<<<<<< Updated upstream
    val point: Int,
    val gamesWon: Int,
    val totalGames: Int,
    val level: Int,
    val exp: Int,
) {
    constructor() : this(0, "", "", "", 0, 0, 0, 0, 0)
    constructor(
        userId: Int,
        id: String,
        nickname: String,
        point: Int,
        gamesWon: Int,
        totalGames: Int,
        level: Int,
        exp: Int
    ) : this(userId, id, "", nickname, point, gamesWon, totalGames, level, exp)
=======
    val email: String? = null
){
    constructor() : this("", "", "")
    constructor(id: String, pass: String) : this(id, pass,"")
>>>>>>> Stashed changes
}